tests/api2/test_308_kubernetes_backup_chart_releases.py

```diff
@@ -0,0 +1,117 @@
+import os
+import pytest
+import sys
+from pytest_dependency import depends
+apifolder = os.getcwd()
+sys.path.append(apifolder)
+from functions import GET, POST, DELETE, wait_on_job
+from auto_config import ha, scale, dev_test
+
+if dev_test:
+    reason = 'Skip for testing'
+else:
+    reason = 'Skipping test for HA' if ha else 'Skipping test for CORE'
+# comment pytestmark for development testing with --dev-test
+pytestmark = pytest.mark.skipif(ha or not scale or dev_test, reason=reason)
+
+
+def test_01_get_plex_version():
+    global plex_version
+    results = POST('/catalog/items/', {'label': 'OFFICIAL'})
+    plex_version = list(results.json()['charts']['plex']['versions'].keys())[0]
+
+
+@pytest.mark.dependency(name='release_plex')
+def test_02_create_plex_chart_release(request):
+    depends(request, ['setup_kubernetes'], scope='session')
+    global release_id
+    payload = {
+        'catalog': 'OFFICIAL',
+        'item': 'plex',
+        'release_name': 'myplex',
+        'train': 'charts',
+        'version': plex_version
+    }
+    results = POST('/chart/release/', payload)
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    job_status = wait_on_job(results.json(), 300)
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+    release_id = job_status['results']['result']['id']
+
+
+@pytest.mark.dependency(name='ix_app_backup')
+def test_03_create_kubernetes_backup_chart_releases_for_ix_applications(request):
+    depends(request, ['release_plex'])
+    global backup_name
+    results = POST('/kubernetes/backup_chart_releases/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    job_status = wait_on_job(results.json(), 300)
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+    backup_name = job_status['results']['progress']['description'].split("'")[1]
```

Comment:
Let's please use the returned value for determining the backup name used ( https://github.com/truenas/middleware/pull/6628 )

Reply:
Done